Contribution.md

```diff
@@ -0,0 +1,153 @@
+# Contribution guidelines
+The guidelines for contribution to babitz backend project are listed below. Please go through it before any contribution.
+
+## 1. General guidelines
+- The general guidelines for writing code. We will be using a subset of [typescript contribution guidelines](https://github.com/microsoft/TypeScript/wiki/Coding-guidelines). In case any of these conflict with loopback4 guidelines, loopback4's guideline is taken as priority. 
+
+  ### Names
```

Comment:
Can you link to the respective sections of the TypeScript contribution guidelines.

Reply:
done.